### PR TITLE
Sophia - adding Rider Edit Page

### LINKED ISSUE #40: add Rider Edit Page
Add a Rider Edit Page to meet acceptance criteria for #7

### PR DESCRIPTION
**Pull request by SophiaWeiler**
In this PR, I have added an EditPage for the ride request + a storybook component + test file. However, because it takes input from the backend to fill the values of form for the RiderEditPage, the storybook component does not render what it will actually look like. I do not anticipate this being an issue though when it is deployed since it passes the tests that mock the backend filling the form values. Helps work towards finishing #7

### CODE REVIEW

**Review by zachmiller-ucsb — CHANGES_REQUESTED** [review]
I can only see the rider create page in storybook?

**Comment by SophiaWeiler** [comment]
> I can only see the rider create page in storybook?

I made a note about this in the PR description, because it takes input from the backend, it doesn't render a storybook page properly. This was also the case for team03, but when deployed with a functioning backend, it rendered as expected.

**Comment by mike-k999** [comment]
> 

I think Sophia is correct, I remembere something like this happening in team03.  I see the form on storybook at least.
EDIT: I tried to test it locally and view the page but I am getting an error.  All I did was include the page into App.js and try to view it, maybe this is because it is missing the backend?
I'm getting these messages?
Screenshot

**Review by zachmiller-ucsb — DISMISSED** [review]
Michael and I tried to test locally and it wasn't working for us. Maybe tomorrow or Tuesday we can rework things

**Review by pconrad — CHANGES_REQUESTED** [review]
Please check the naming... a lot of things that have `Rider` on them should probably have `RideRequest` on them.

A `Rider` is a person.  We would expect a  `RiderForm` to be a form for entering information about a person, not about a Ride.    For that we want a `RideForm` or a `RideRequestForm`.

**Comment by SophiaWeiler** [comment]
> Michael and I tried to test locally and it wasn't working for us. Maybe tomorrow or Tuesday we can rework things

I think it is because we don't have a working backend so there is nothing to GET from.

File: frontend/src/main/pages/Ride/RideRequestEditPage.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+
+  const objectToAxiosPutParams = (ride) => ({
+    url: "/api/ride_request",
+    method: "PUT",
+    params: {
+      id: ride.id,
+    },
+    data: {
+        riderId: ride.riderId,
```
Comment (strongani) [inline frontend/src/main/pages/Ride/RideRequestEditPage.js]: Put for backend does not require riderId, automatically fills it in. So, it should not have riderId in params. Also for tests.

**Review by strongani — DISMISSED** [review]
Remove riderId parameter from api requests.

**Review by mike-k999 — DISMISSED** [review]
lgtm

**Review by pconrad — APPROVED** [review]

**Pull request merged**